### PR TITLE
[WIN32KNT_APITEST] Improve NtGdiSetBitmapBits 6

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

JIRA issue: [CORE-15657](https://jira.reactos.org/browse/CORE-15657) and [CORE-15695](https://jira.reactos.org/browse/CORE-15695)

- Add some PAGE_READONLY tests.
- Add two bit pattern tests.

### CODE REVIEW

**Comment by katahiromz** [comment]
Evidence:
evidence

**Comment by katahiromz** [comment]
Cc @ThFabba

File: modules/rostests/apitests/win32nt/ntgdi/NtGdiSetBitmapBits.c [hunk 1]
```diff
@@ -88,7 +88,26 @@ START_TEST(NtGdiSetBitmapBits)
...
+    SetLastError(0xDEADFACE);
+    ok_long(NtGdiSetBitmapBits(hBitmap, 0x100001, LargeBits), 0x0);
+    ok_long(GetLastError(), 0xDEADFACE);
+
+    VirtualFree(LargeBits, 0, MEM_RELEASE);
+
+    /* Test large byte counts (with PAGE_READONLY) */
+    LargeBits = VirtualAlloc(NULL, 0x100000 + PAGE_SIZE, MEM_RESERVE, PAGE_READONLY);
+    VirtualAlloc(LargeBits, 0x100000, MEM_COMMIT, PAGE_READWRITE);
+    CopyMemory(LargeBits, Bits, sizeof(Bits));
```
Comment (ThFabba) [inline modules/rostests/apitests/win32nt/ntgdi/NtGdiSetBitmapBits.c]: I would assume this is going to crash...

File: modules/rostests/apitests/win32nt/ntgdi/NtGdiSetBitmapBits.c [hunk 2]
```diff
@@ -88,7 +88,26 @@ START_TEST(NtGdiSetBitmapBits)
...
+
+    SetLastError(0xDEADFACE);
+    ok_long(NtGdiSetBitmapBits(hBitmap, 0x100, LargeBits), 0xC);
```
[the remark this answers is not in the document]
Comment (katahiromz) [reply modules/rostests/apitests/win32nt/ntgdi/NtGdiSetBitmapBits.c]: The code for `0x100001` you say is below. The same code is omitted.

**Pull request merged**